code/server/src/ServerModule.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * author: Elisa Maiettini
+ * date: 26/11/2016
+ */
+#include<ServerModule.hpp>
+
+using namespace std;
+using namespace yarp::os;
+
+bool ServerModule::respond(const Bottle &command, Bottle &reply){
+	int tested_N;
+	cout << "ServerModule: waiting message from client.."<< endl;
+
+	if(command.get(0).asInt()==COLLATZ_VOCAB_REQ_ITEM){
```

Comment:
nothing because it means that it isn't a correct message. I can add an 'else' to print the error